.vscode/settings.json

```diff
@@ -24,8 +24,8 @@
     "**/*.js": true
   },
   "editor.formatOnSave": true,
-  "prettier.stylelintIntegration": true,
-  "prettier.eslintIntegration": true,
+  "prettier-stylelint.stylelintIntegration": true,
```

Comment:
cc @cartogram I had to update these two to get prettier working.
![Screen Shot 2019-11-11 at 10 27 37 AM](https://user-images.githubusercontent.com/1610169/68599689-029dd300-046f-11ea-8749-8b7c7673b9e2.png)

double checking on if this is related to eslint update you did recently / if there are anything better to do

Reply:
Make sense to me 👍